packages/jobs/lib/temporal.ts

```diff
@@ -46,7 +46,8 @@ export class Temporal {
             namespace: this.namespace,
             workflowsPath: createRequire(import.meta.url).resolve('./workflows'),
             activities,
-            taskQueue: TASK_QUEUE
+            taskQueue: TASK_QUEUE,
+            maxConcurrentWorkflowTaskExecutions: 50
```

Comment:
I picked 50 because normal volume of concurrent workflows in temporal seems to be around 20 every time I check